### PR TITLE
add login and registration functionality

### PR DESCRIPTION
**Pull request by 00ec454**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -10,7 +10,7 @@ buildscript {
         maven { url 'https://maven.fabric.io/public' }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.3'
+        classpath 'com.android.tools.build:gradle:3.0.0-beta7'
```
Comment (m3libea) [inline build.gradle]: I modified that line in order to gradle work with the Stable Android studio (2.3.3) That makes reference to he new one.

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#Mon Oct 02 20:56:36 PDT 2017
+#Thu Oct 12 21:29:57 PDT 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
```
Comment (m3libea) [inline gradle/wrapper/gradle-wrapper.properties]: Same here, with that version the project doesn't work on Android Studio 2.3.3 (I also changed that line in the downgrade)

File: build.gradle (hunk 1 above)
Comment (ghilbing) [reply build.gradle]: great!

Comment (00ec454) [reply build.gradle]: yah next time I make sure dont check in this file

File: gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (00ec454) [reply gradle/wrapper/gradle-wrapper.properties]: same won't checking in future

File: build.gradle (hunk 1 above)
Comment (m3libea) [reply build.gradle]: Perfect :)

**Pull request merged**